bower.json

```diff
@@ -25,7 +25,8 @@
     "nunjucks": "nunjucks#1.0.5",
     "nwmatcher": "nwmatcher#1.3.4",
     "pikaday": "~1.3.2",
-    "react": "#0.13.0",
+    "react": "#0.13.3",
```

Comment:
Aha, actually, I just tried going back to 0.13.0 and bower complained that ``react-swipe-views`` require ``~0.13.1`` which resolves to 0.13.3.